application/routes.py

```diff
@@ -1,30 +1,22 @@
 """ Defines all Routes """
-from .base import add_resource, add_method
 
 
-# from flask_restful import reqparse, abort, Api, Resource
-
-
-def create_routes(api):
+def create_routes(app):
     """ Creates Routes. Called in __init__ """
 
-    from .controllers.home_controller import HomeController
-    from .controllers.artifacts_controller import ArtifactsController
-    from .controllers.tags_controller import TagsController
-    from .controllers.presentations_controller import PresentationsController
-    from .controllers.teams_controller import TeamsController
-    from .controllers.users_controller import UsersController
-    from .controllers.authentications_controller import AuthenticationsController
+    from application.home.home_view import HomeView
+    from application.artifacts import ARTIFACTS
+    from application.users import USERS
+    from application.teams import TEAMS
+    from application.authentications import AUTHENTICATIONS
+    from application.presentations import PRESENTATIONS
+    from application.artifacts.tags.tags_view import TagsView
+
+    app.register_blueprint(ARTIFACTS, url_prefix='/images')
+    app.register_blueprint(USERS, url_prefix='/users')
+    app.register_blueprint(TEAMS, url_prefix='/teams')
+    app.register_blueprint(AUTHENTICATIONS, url_prefix='/authentications')
+    app.register_blueprint(PRESENTATIONS, url_prefix='/presentations')
 
-    add_method(api, '/', "show_status", HomeController, method="get")
-    add_resource(api, '/images', ArtifactsController)
-    add_method(api, '/images', "update_many", ArtifactsController, method="patch")
-    add_resource(api, '/teams', TeamsController)
-    add_resource(api, '/users', UsersController)
-    add_resource(api, '/authentications', AuthenticationsController, only=["create"])
-    add_method(api, '/authentications', "delete", AuthenticationsController, method="delete")
-    add_resource(api, '/presentations', PresentationsController, only=["create"])
-    add_method(api, '/images/<object_id>/tags', "add_tags",
-               TagsController, method="post")
-    add_method(api, "/tags/suggested", "suggested_tags",
-               TagsController, method="get")
+    app.add_url_rule('/tags/suggested', "suggested_tags", TagsView().suggested_tags)
```

Comment:
I think we should move that to teams btw, but not in that PR.

Reply:
Yes and move teams to users imo.